js/backboneAgent/controllers/appViewController.js

```diff
@@ -151,6 +159,47 @@ Modules.set('controllers.appViewController', function() {
 
                 return result;
             }});
+        },
+
+        setupHighlight: function() {
+            this.highlightMask = document.createElement('div');
+            this.highlightMask.style.position = 'absolute';
+            this.highlightMask.style.zIndex = '100000000000';
+            this.highlightMask.style.pointerEvents = 'none';
+            this.highlightMask.style.backgroundColor = 'rgba(55, 161, 243, 0.48)';
+            this.highlightMask.style.webkitFilter = 'grayscale(20%)';
+
+            // unhighlight when the user closes the devtools, etc.
+            this.listenTo(port, 'client:disconnect', this.unhighlightViewElements);
```

Comment:
This code is executed by the agent, which is separate from the devtools panel, runs in background from when you reload the app in "debug mode" until you don't refresh the page. You can also close the devtools and reopen it, it will be still there and the panel will refetch the captured info.

So I don't think there is the need to remove it, it is a single dom element that is created just one time and then is moved under the element to highlight.